Gemfile

```diff
@@ -1,14 +1,14 @@
 source 'https://rubygems.org'
```

Comment:
Lets clean this file up now by moving the mysql2 and jdbc-mysql accordingly and removing any unused gems.

Thanks.